app/api/books/v1/__init__.py

```diff
@@ -0,0 +1,10 @@
+from flask import Blueprint
+from flask_restful import Resource,Api
+#local imports
+from app.api.books.v1.views import AddBook
+
+bk1 = Blueprint('book_v1',__name__,url_prefix='/api/v1')
```

Comment:
I feel `bk1` is a bit ambiguous, why not something like `library_v1`

Reply:
let me change it